### PR TITLE
Fstab agent test

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: library/general/test/agents_test/fstab_agent_test.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      expect(content).to satisfy { |r| r.find { |e| e["file"] == "/media/new2" } }
+    end
+
+    it "returned array contains tmpfs entry" do
+      expect(content).to satisfy { |r| r.find { |e| e["file"] == "/tmp" } }
+    end
+
+    it "see comments" do
+      expect(content).to include("#comment")
+      pending "need to be fixed"
```
Comment (kobliha) [inline library/general/test/agents_test/fstab_agent_test.rb]: Did you want to write "# my fine comment"?

Comment (jreidinger) [inline library/general/test/agents_test/fstab_agent_test.rb]: actually it is POC to see error, I do not know how anyagent return comments

File: library/general/test/agents_test/fstab_agent_test.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env rspec
+
+require_relative "../test_helper"
+require "yast"
+
+describe ".proc.meminfo" do
+  around :each do |example|
```
Comment (imobachgs) [inline library/general/test/agents_test/fstab_agent_test.rb]: The `:each` argument is not needed, right?

File: library/general/test/agents_test/fstab_agent_test.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    root = File.join(File.dirname(__FILE__), "test_root")
```
Comment (imobachgs) [inline library/general/test/agents_test/fstab_agent_test.rb]: Could it be a constant?

File: library/general/test/agents_test/fstab_agent_test.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  describe ".Read" do
+    let(:content) { Yast::SCR.Read(path(".etc.fstab")) }
+
+    it "reads content of /etc/fstab and returns array" do
+      expect(content).to be_a(Array)
+    end
+
+    it "returned array contains nfs entries" do
```
Comment (teclator) [inline library/general/test/agents_test/fstab_agent_test.rb]: returned? maybe "returns an array of nfs entries" or containing ..

Comment (jreidinger) [inline library/general/test/agents_test/fstab_agent_test.rb]: right, will change it

File: library/general/test/agents_test/fstab_agent_test.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      expect(content).to satisfy { |r| r.find { |e| e["file"] == "/home/kv2" } }
+      expect(content).to satisfy { |r| r.find { |e| e["file"] == "/media/new" } }
+      expect(content).to satisfy { |r| r.find { |e| e["file"] == "/media/new2" } }
+    end
+
+    it "returned array contains tmpfs entry" do
+      expect(content).to satisfy { |r| r.find { |e| e["file"] == "/tmp" } }
+    end
+
+    it "see comments" do
```
Comment (teclator) [inline library/general/test/agents_test/fstab_agent_test.rb]: "is able to parse comments"

File: library/general/test/agents_test/fstab_agent_test.rb (hunk 4 above)
Comment (imobachgs) [inline library/general/test/agents_test/fstab_agent_test.rb]: np: I would rephrase as "returns an array containing NFS entries"

File: library/general/test/agents_test/fstab_agent_test.rb (hunk 5 above)
Comment (jreidinger) [inline library/general/test/agents_test/fstab_agent_test.rb]: well, according to any agant doc it should not even parse it...but question is how to write it back and maybe there is no way, so it is proper behavior :) Welcome to ancient yast

File: library/general/test/agents_test/fstab_agent_test.rb [hunk 6]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env rspec
+
+require_relative "../test_helper"
+require "yast"
+
+describe ".proc.meminfo" do
```
Comment (mvidner) [inline library/general/test/agents_test/fstab_agent_test.rb]: @jreidinger please fix to `.etc.fstab` in the next PR
